src/main/groovy/au/org/ala/ecodata/converter/SciStarterConverter.groovy

```diff
@@ -187,9 +162,7 @@ class SciStarterConverter {
                                     UN_region.toLowerCase() == region.toLowerCase()
                                 }
 
-                                if (matchedRegion) {
-                                    matchedRegions.push(matchedRegion)
-                                }
+                                if (matchedRegion) matchedRegions.push(matchedRegion)
```

Comment:
I actually prefer the previous formatting than the new here - we probably should add a formatter to the project.

Reply:
Reverted the formatting